### PR TITLE
Toogle workout card

### PR DESCRIPTION
**Pull request by BerePendzialek**
Hi, here is my 2. User story. The workout-card has now a button that shows/hides the description of each section of a workout. I also added storybook and some tests. Style is very minimal. Enjoy and thanks in advance for your comments.

### CODE REVIEW

File: src/components/WorkoutCard/WorkoutCard.spec.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import WorkoutCard from './WorkoutCard'
 import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Button from '../Button'
```
Comment (macfrei) [inline src/components/WorkoutCard/WorkoutCard.spec.js]: I wouldn't import the Button here as you then only test the Button and not the WorkoutCard. You should find the button in your WorkoutCard component instead.

File: src/components/WorkoutCard/WorkoutCard.spec.js [hunk 2]
```diff
@@ -33,12 +35,20 @@ const props = {
...
     render(<WorkoutCard {...props} />)
     expect(screen.getByText(/On FTP-Light/i)).toBeInTheDocument()
     expect(
       screen.getByText(/Ride right on FTP for a period of time/i)
     ).toBeVisible()
     expect(screen.getByText(/40/)).toBeInTheDocument()
+    expect(screen.getByRole('button')).toBeInTheDocument()
+  })
+
+  it('has a button that calls onClick prop when clicked', () => {
```
Comment (macfrei) [inline src/components/WorkoutCard/WorkoutCard.spec.js]: I think you want to test whether the section is visible after a button click or not. For that you will need to render your WorkoutCard, find the Button in there, click it and then check whether the section is visible or not. You don't need a callback function then. If you have a questions about it, please ask :)

**Review by macfrei — COMMENTED** [review]
Nice implementation! I added a few comments 🚀

File: src/components/WorkoutCard/WorkoutCard.spec.js (hunk 1 above)
Comment (BerePendzialek) [reply src/components/WorkoutCard/WorkoutCard.spec.js]: Thanks! I added the changes

**Pull request merged**